File: Selenium/Activities/Activity_13.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Chrome() as driver:
    driver.get("https://v1.training-support.net/selenium/tables")
    print("The page title is: ", driver.title)
    rows = driver.find_elements(By.XPATH, "//table[@class='ui celled striped table']/tbody/tr")
    columns = driver.find_elements(By.XPATH, "//table[@class='ui celled striped table']/tbody/tr[1]/td")
    print("Number of row of first table is", len(rows))
    print("Number of column of first table is", len(columns))
    third_row_ele_list = driver.find_elements(By.XPATH, "//table[@class='ui celled striped table']/tbody/tr[3]/td")
    for ele in third_row_ele_list:
        print(ele.text)

    print("Second row second column value is: ", driver.find_element(By.XPATH, "//table[@class='ui celled striped table']/tbody/tr[2]/td[2]").text)
    driver.quit()

File: Selenium/Activities/Activity_13.java
package Activities;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;

import java.util.List;

public class Activity_13 {
    public static void main(String[] args) {
        WebDriver driver = new ChromeDriver();
        driver.get("https://v1.training-support.net/selenium/tables");
        System.out.println("The pagetitle is: " + driver.getTitle());
        List<WebElement> row = driver.findElements(By.xpath("//table[@class='ui celled striped table']/tbody/tr"));
        List<WebElement> column = driver.findElements(By.xpath("//table[@class='ui celled striped table']/tbody/tr[1]/td"));
        System.out.println("The number of rows: " + row.size());
        System.out.println("The number of columns: " + column.size());
        List<WebElement> third_row_cell_ele = driver.findElements(By.xpath("//table[@class='ui celled striped table']/tbody/tr[3]/td"));
        for(WebElement ele:third_row_cell_ele){
            System.out.println(ele.getText());
        }
        System.out.println("Second row second column value is: " + driver.findElement(By.xpath("//table[@class='ui celled striped table']/tbody/tr[2]/td[2]")).getText());
        driver.quit();
    }
}
